app.js

```diff
@@ -14,7 +16,10 @@ if (name) {
   alert(name + ', huh? Interesting, is that a family name or something? Well alright then ' + name + ', it\'s truly time to begin! This is a guessing game! Can you guess what you\'ll be doing? Answer five questions in Y or N format, but beware!! Invalid inputs can yield rather... dangerous results. Mwahahahaha!');
 }
 
+function question1Function() {
+
 var question1 = prompt('Do I have a dog? Answer Y or N to submit a valid response. Like, seriously ' + name + ', don\'t type some random bullshit.').toUpperCase();
```

Comment:
watch you indentation here, everything inside the function should be indented a couple of spaces.